config/services.yml

```diff
@@ -397,6 +397,8 @@ https://cmr.earthdata.nasa.gov:
         bbox: false
         shape: false
         temporal: false
+        variable: true
+        multiple_variable: false
```

Comment:
Just FYI that we don't do anything with this information yet. I can write a ticket to make it so services don't match to a request if they request multiple variables and the service supports variable, but not multiple variable.